test/renate/data/test_data_module.py

```diff
@@ -91,15 +91,15 @@ def test_torchvision_data_module(tmpdir, dataset_name, num_tr, num_te, x_shape):
 @pytest.mark.parametrize(
     "dataset_name,chunk_id,num_tr,num_te",
     [
-        ("CLEAR10", 0, 2986, 500),
-        ("CLEAR100", 0, 9945, 4984),
+        ("CLEAR10", 0, 3300, 550),
+        ("CLEAR100", 0, 9964, 5000),
```

Comment:
did you verify that the first number is not 10K but 9964?

Reply:
Yes.